src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationController.java

```diff
@@ -51,15 +55,21 @@ public OcrValidationController(
     })
     public ResponseEntity<OcrValidationResponse> validateOcrData(
         @RequestHeader(name = "ServiceAuthorization", required = false) String serviceAuthHeader,
+        @PathVariable(name = "form-type", required = false) String formType,
         @Valid @RequestBody OcrDataValidationRequest request
     ) {
+        if (!Enums.getIfPresent(FormType.class, formType).isPresent()) {
+            throw new FormNotFoundException("Form type not found " + formType);
+        }
+
         String serviceName = authService.authenticate(serviceAuthHeader);
         logger.info("Request received to validate ocr data from service {}", serviceName);
 
         authService.assertIsAllowedService(serviceName);
 
-        OcrValidationResult result = ocrDataValidator.validate(request.getFormType(), request.getOcrDataFields());
+        OcrValidationResult result = ocrDataValidator.validate(FormType.PERSONAL, request.getOcrDataFields());
```

Comment:
I hope `FormType.PERSONAL` can be exchanged to actual request path value can it not?

Reply:
yeah.. I added when I was trying to fix a test but forgot to remove it. thanks @doncem

Reply:
Updated